lib/diagnostic.js

```diff
@@ -28,7 +35,11 @@
 // Attributes: name, address, numberOfPets
 
 // Replace null with your answer
-let owner = null;
+let owner = {
+  name: 'J Beebs',
+  address: '55 Nutball Lane';
+  numberOfPets: 8
```

Comment:
Watch your syntax - linter should have indicated in some way that there shouldn't be a semicolon after ` address: '55 Nutball Lane';` (should instead be a comma)